docs/en/observability/slo-privileges.asciidoc

```diff
@@ -0,0 +1,95 @@
+[[slo-privileges]]
+= Configure service-level objective (SLO) access 
+
+++++
+<titleabbrev>Configure SLO access</titleabbrev>
+++++
+
+beta::[]
+
+You can create the following roles for your SLOs:
+
+* <<slo-all-access,*SLO All*>> — Create, delete, and see SLOs and their historical summaries.
+* <<slo-read-access,*SLO Read*>> — See SLOs and their historical summaries.
+
+To create a role:
+
+. From the left navigation in {kib}, under *Management* select *Stack Management*
+. Under *Security*, select *Roles*
+. Click *Create role* in the upper-right corner of the page.
+
+[discrete]
+[[slo-all-access]]
+== Create an SLO All role
+
+Set the following privileges for the SLO All role:
+
+[cols="1,1"]
+|===
+|Group|Action
+
+|Cluster privileges
+|Add `manage_index_template`, `manage_ilm`, `manage_pipeline`, and `manage_transform` to the *Cluster Privileges* field.
+
+|Index privileges
+|Add `*` to the *Indices* field and `all` to the *Privileges* field.
```

Comment:
Sorry I was not specific enough here, the index privileges can be restricted to the index pattern: `.slo-*` with `all` privileges.